modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2DeveloperKeysForDXP.testcase

```diff
@@ -121,4 +121,266 @@ definition {
 		}
 	}
 
+	@description = "Automation ID: LPS-166936 | Test Summary: Verify that the user with the Admin and Partner Manager roles can see the actions (Generate New - Deactivate - Actions Menu - Deactivate button)"
+	@ignore = "true"
+	@priority = "4"
+	test UserAdminAndPartnerManagerRoleCanSeeAndPerformAllActionsForDXP {
+
+		// Needs refactor since it depends on example Avangrid Portals account
+
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		}
+
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "${testSiteName}");
+		}
+
+		task ("Change the user role in the project") {
+			Account.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "NASA ICE");
+
+			Account.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+
+			Account.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Partner Manager",
+				accountUserName = "Test Test");
```

Comment:
Pablo, we are adding here two Roles for the same user and just after that we check the visibility of the buttons. An error may be hidden because the user has both roles. Shouldn't we test separately, so we can guarantee the role is working as expected?